### DISCUSSION TITLE
Launch Chromium with Hardware Acceleration disabled

### DISCUSSION THREAD

**Discussion by garysassano** [jef/streetmerchant#2140] [General]
Is there a way to disable Hardware Acceleration for the Chromium browser instances?

**Comment by Found0415** [comment]
I have my HE turned off by manually opening Chromium and turning it off in the settings. Does it switch back on for you in each instance?

**Reply by garysassano** [reply]
Yes, every instance created automatically by streetmerchant has HA turned on by default. I think there must be a way to change the settings of the browser permanently, but I don't know how. Alternatively, I think it should be possible to add a parameter to the executable so that it's started with HA disabled.
